R/wwinference.R

```diff
@@ -138,7 +135,7 @@
 #'   fit_opts = get_mcmc_options(
 #'     iter_warmup = 250,
 #'     iter_sampling = 250,
-#'     n_chains = 2
+#'     chains = 2
```

Comment:
This section needs revision given the changes to `fit_opts`

Good opportunity to review the whole example and try running it (since it isn't run for the docs due to the `\dontrun` command)